applications/RANSModellingApplication/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+    ${CMAKE_CURRENT_SOURCE_DIR}/rans_modelling_application_variables.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/rans_calculation_utilities.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/evm_k_epsilon/evm_k_epsilon_utilities.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/evm_k_epsilon/rans_evm_k_element.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/evm_k_epsilon/rans_evm_low_re_k_element.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/evm_k_epsilon/rans_evm_epsilon_element.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/evm_k_epsilon/rans_evm_low_re_epsilon_element.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_strategies/relaxed_dof_updater.cpp
+)
+
+set( KRATOS_RANS_MODELLING_APPLICATION_PYTHON_INTERFACE
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/rans_modelling_python_application.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_utilities_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_strategies_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_strategies_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_processes_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_auxiliary_processes_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_solving_processes_to_python.cpp
+)
+
+set(KRATOS_RANS_MODELLING_APPLICATION_SOURCES_MPI "")
+set(KRATOS_RANS_MODELLING_APPLICATION_EXTRA_LIBS "")
+if(${MPI_NEEDED} MATCHES ON )
+    include_directories( ${CMAKE_SOURCE_DIR}/applications/TrilinosApplication )
+    find_package(TRILINOS REQUIRED)
+    include_directories(SYSTEM ${TRILINOS_INCLUDE_DIR}) # Included as "SYSTEM" to suppress warnings from it
+
+    set(KRATOS_RANS_MODELLING_APPLICATION_EXTRA_LIBS ${KRATOS_RANS_MODELLING_APPLICATION_EXTRA_LIBS} ${MPI_LIBRARIES})
+    set(KRATOS_RANS_MODELLING_APPLICATION_EXTRA_LIBS ${KRATOS_RANS_MODELLING_APPLICATION_EXTRA_LIBS} ${TRILINOS_LIBRARIES})
+endif(${MPI_NEEDED} MATCHES ON )
+
+if(${KRATOS_BUILD_TESTING} MATCHES ON)
+  file(GLOB_RECURSE
+    KRATOS_RANS_MODELLING_APPLICATION_TESTING
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/cpp/*.cpp
+  )
+endif(${KRATOS_BUILD_TESTING} MATCHES ON)
+
+add_library(KratosRANSModellingCore SHARED ${KRATOS_RANS_MODELLING_APPLICATION_CORE} ${KRATOS_RANS_MODELLING_APPLICATION_TESTING})
+target_link_libraries(KratosRANSModellingCore PUBLIC KratosCore KratosFluidDynamicsCore ${KRATOS_RANS_MODELLING_APPLICATION_EXTRA_LIBS})
+set_target_properties(KratosRANSModellingCore PROPERTIES COMPILE_DEFINITIONS "RANS_MODELLING_APPLICATION=EXPORT,API")
+
+pybind11_add_module(KratosRANSModellingApplication MODULE THIN_LTO ${KRATOS_RANS_MODELLING_APPLICATION_PYTHON_INTERFACE})
+target_link_libraries(KratosRANSModellingApplication PRIVATE KratosRANSModellingCore)
+set_target_properties(KratosRANSModellingApplication PROPERTIES PREFIX "")
+
+if(USE_COTIRE MATCHES ON)
+    cotire(KratosRANSModellingApplication)
```

Comment:
don't you also have to cotire `KratosRANSModellingCore`? I think you should

Reply:
done